.github/workflows/scala.yml

```diff
@@ -0,0 +1,26 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'sbt'
+      - name: Assemble JARs
+        run: sbt assembly
```

Comment:
Should run tests as well?

Reply:
There are some broken unit tests related to fork detection which will be time consuming to fix. I am therefore leaving it for the next release.